cli/cmdlineparser.cpp

```diff
@@ -98,12 +98,11 @@ static bool addPathsToSet(const std::string& fileName, std::set<std::string>* se
 
 CmdLineParser::CmdLineParser(Settings *settings)
     : mSettings(settings)
-    , mShowHelp(false)
-    , mShowVersion(false)
-    , mShowErrorMessages(false)
-    , mExitAfterPrint(false)
-{
-}
+      , mShowHelp(false)
```

Comment:
hmm.. can this comma have the same indentation as the colon?

Reply:
Yes, but if we use trailing commas then the variables will be aligned to the `:` instead of variable name.

Reply:
ok that is unfortunate.. but maybe we can adjust the source files..

Reply:
So I changed the `indent_constr_colon` setting to `false` to get it to align with leading commas. You can tweak this and the source files to get the style you want. There might be a way to enforce trailing commas, but I am not sure.